### PR TITLE
provider/docker: Send proper event details to echo.

### PR DESCRIPTION
**Pull request by lwander**
I was incorrectly using the `details` field of the event data sent to Echo rather than `content`. 

I also switched formatting of `imageId` from `$registry/$repository:$tag` to the image key stored in redis, for simplicity.

@duftler

### CODE REVIEW

File: igor-web/src/main/groovy/com/netflix/spinnaker/igor/docker/DockerMonitor.groovy [hunk 1]
```diff
@@ -32,6 +32,7 @@ import rx.Scheduler
 import rx.functions.Action0
 import rx.schedulers.Schedulers
 
+import javax.print.Doc
```
Comment (duftler) [inline igor-web/src/main/groovy/com/netflix/spinnaker/igor/docker/DockerMonitor.groovy]: s/javax.print.Doc/javax.swing.plaf.metal.OceanTheme

File: igor-web/src/main/groovy/com/netflix/spinnaker/igor/docker/DockerRegistryCache.groovy [hunk 2]
```diff
@@ -38,14 +39,7 @@ class DockerRegistryCache {
...
 
-        def res = resource.keys(key).collect {
-            def parse = parseKey(it)
-            if (parse) {
-                formatImage(parse.registry, parse.repository, parse.tag)
-            } else {
-                null
-            }
-        }.findAll { it != null }
+        List<String> res = resource.keys(key).findAll { it != null } as List
```
Comment (duftler) [inline igor-web/src/main/groovy/com/netflix/spinnaker/igor/docker/DockerRegistryCache.groovy]: Could also use:

`res = resource.keys(key).findResults( { it } )`

or

`res = resource.keys(key) - null`

**Comment by duftler** [comment]
LGTM after addressing comments.

File: igor-web/src/main/groovy/com/netflix/spinnaker/igor/docker/DockerRegistryCache.groovy (hunk 2 above)
Comment (tomaslin) [inline igor-web/src/main/groovy/com/netflix/spinnaker/igor/docker/DockerRegistryCache.groovy]: or resource.keys(key).removeAll([null]) which might be slightly more readable

**Pull request merged**